tests/main/auto-refresh-gating/task.yaml

```diff
@@ -0,0 +1,157 @@
+summary: Check that auto-refresh with gate-auto-refresh hooks works.
+
+details: |
+  Test auto-refresh with gate-auto-refresh hook support enabled
+  (experimental.gate-auto-refresh-hook feature) and verify the hook can control
+  automatic refreshes. The test uses two test snaps, one of them
+  being a content provider of the other. There are a few versions of these
+  snaps in the store (in stable/beta/edge channels) for this test.
+
+environment:
+  SNAP_NAME: test-snapd-refresh-control
+  CONTENT_SNAP_NAME: test-snapd-refresh-control-provider
+  CONTROL_FILE: /var/snap/test-snapd-refresh-control/common/control
```

Comment:
You could have this var as well in order to simplify the test
DEBUG_LOG_FILE: /var/snap/test-snapd-refresh-control/common/debug.log

Reply:
Done.